src/lexer.h

```diff
@@ -35,6 +35,6 @@ int getType(char *);
 Token *getTokens(char *);
 int isunderscore(char);
 int isValidToken(char *);
-int findLongestToken(char *, Token *, int *);
+void findLongestToken ( char* token, Token* tokArr, int* tokenCount );
```

Comment:
Please follow the same pattern as followed before while declaring pointers like `char *token`. Helps maintain consistency